src/components/UniversalNavbarExpanded/MobileNav/AccordionNav.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+
+// Reused assets from UniversalNavbar
+import { AccordionToggleIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { TitleMedium, TitleSmall } from '../../index'
+
+// Parent component (UniversalNavbar) siblings
+import NavLink from '../NavLink'
+
+//Helpers
+import { isEnterKeyPress } from '../../../helpers/isEnterKeyPress'
+
+// Styles
+import styles from './AccordionNav.module.scss'
+
+/**
+ * Interactive accordion menu, displays at top of Mobile Nav.
+ *
+ * TODO should the accordion collapse an open section when the parent modal is dismissed?
```

Comment:
Side effect of reducing the amount of re-rending, the previously expanded section remains open when toggling the mobile nav closed.

Reply:
a78d6e1